en/mavlink_generator.md

```diff
@@ -0,0 +1,17 @@
+# MAVLink Generator (C/C++, Python)
+
+MAVLink is distributed with a common set of messages. Custom messages can be generated and included as a replacement for the common message set or as extension to it. See the [minimal.xml](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/minimal.xml) for a minimal example and [uAvionix.xml](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/uAvionix.xml) for an example of using the common message set with some project-specific extensions.
+
+Download the generator from here: https://github.com/mavlink/mavlink
```

Comment:
How do I "download the generator"? - or does this just mean "got to this location"?

When I try the following command to obtain the genetor GUI I get:
```
D:\Github\forks\mavlink\mavlink>python mavgenerate.py
Traceback (most recent call last):
  File "mavgenerate.py", line 42, in <module>
    from pymavlink.generator import mavgen
ModuleNotFoundError: No module named 'pymavlink.generator'
````
I installed pymavlink (`pip install pymavlink`) and now I get:
```
D:\Github\forks\mavlink\mavlink>python mavgenerate.py
Traceback (most recent call last):
  File "mavgenerate.py", line 205, in <module>
    app = Application()
  File "mavgenerate.py", line 54, in __init__
    self.createWidgets()
  File "mavgenerate.py", line 126, in createWidgets
    self.strict_units_value.set(mavgen.DEFAULT_STRICT_UNITS)
AttributeError: module 'pymavlink.generator.mavgen' has no attribute 'DEFAULT_STRICT_UNITS'
```
Next steps?

Reply:
Have you done `git submodule update --init --recursive` ?

Reply:
Yes, same error as reported in https://github.com/mavlink/mavlink/issues/840#issuecomment-366087618:

```
D:\Github\forks\mavlink\test2\mavlink>git submodule update --init --recursive
Submodule 'pymavlink' (https://github.com/mavlink/pymavlink.git) registered for path 'pymavlink'
Cloning into 'D:/Github/forks/mavlink/test2/mavlink/pymavlink'...
error: no such remote ref 5f31f68e6e373336df12d1cea31626097cece6bc
Fetched in submodule path 'pymavlink', but it did not contain 5f31f68e6e373336df12d1cea31626097cece6bc. Direct fetching of that commit failed.
```